libpd_wrapper/z_libpd.h

```diff
@@ -19,50 +19,58 @@ extern "C"
 
 #include "m_pd.h"
 
+#ifndef EXPORT
+# ifdef LIBPD_STATIC
+#  define EXPORT /* Leave blank: friendly with static linking */
```

Comment:
this should be `#define EXPORT extern`. we still need the `extern` keyword for global variables, but we don't want to DLL export/import them.